README.md

```diff
@@ -55,3 +55,9 @@ yarn console
 ```
 yarn test
 ```
+
+## Production
+
+```
+TRUFFLE_PRIVATE_KEY="<wallet-key>" TRUFFLE_RPC="<rpc-endpoint>" yarn deploy --network production
```

Comment:
also maybe add a line saying `truffle migrate`/`truffle deploy` would no longer work since the requirement of environment variables